packages/react-ui/src/widgets/utils/colorUtils.js

```diff
@@ -0,0 +1,25 @@
+/**
```

Comment:
There is another file with color utils, maybe we can merge both files to have only one ¿?

packages/react-ui/src/utils/palette.js